abuild.in

```diff
@@ -1108,6 +1116,11 @@ is_dev_pkg() {
 	test "${subpkgname%-dev}" != "$subpkgname"
 }
 
+# returns true if this is the -static package
+is_static_pkg() {
+	test "${subpkgname%-static}" != "$subpkgname"
```

Comment:
The use of these functions (`is_x_package`) seems to be inconsistent throughout abuild.
I'd personally prefer if we effectively treated them as pure functions (i.e passing `$subpkgname` as an argument) rather than relying on state.

Reply:
Will work on it on a separate PR